index.html

```diff
@@ -46,7 +46,7 @@ <h3 class="link-1"> <a class="link-2" href="#"> LET'S CONNECT</a></h3>
         <section class="work-section">
             <div class="card">
               <div class="res-snapshoot">
-                <img class="Snapshoot" src="./image/Snapshoot-Portfolio.png" alt="img">
+                <img class="Snapshoot" src="./image/Snapshoot-Portfolio.png" alt="Portfolio img one">
```

Comment:
**Suggestions**
In my portfolio, I used an alternative text of null. I thought the message came across clearly even without the images. Some sources suggest this is a good practice.